src/client/components/pages/parts/editor-achievements.js

```diff
@@ -158,6 +150,7 @@ class EditorAchievementTab extends React.Component {
 										const updatedStyle = {
 											...style,
 											background: 'white',
+											borderBottom: '1px solid',
```

Comment:
I find the black border a bit harsh; maybe you could give it the same border color as the surrounding badges (#ebe2df), or some other color?

Reply:
#ebe2df looks subtle and good. Thanks!